crates/router/src/services/encryption.rs

```diff
@@ -50,6 +50,24 @@ impl NonceSequence for NonceGen {
     }
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct JwsBody {
+    pub header: String,
+    pub payload: String,
+    pub signature: String,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct JweBody {
+    pub header: String,
+    pub iv: String,
+    #[serde(rename = "encryptedPayload")]
+    pub encrypted_payload: String,
+    pub tag: String,
+    #[serde(rename = "encryptedKey")]
+    pub encrypted_key: String,
+}
```

Comment:
can we use `#[serde(rename_all = "camelCase")]`?

Reply:
yes